static/css/main.css

```diff
@@ -32,6 +32,13 @@ ul {
   padding-left: 1em;
 }
 
+.gci-logo{
```

Comment:
Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

*Origin: CSSLintBear, Section: `css`.*

Reply:
Also add space before {